rpc/jsonrpc/tracing.go

```diff
@@ -372,7 +372,7 @@ func (api *DebugAPIImpl) TraceCall(ctx context.Context, args ethapi.CallArgs, bl
 	if err != nil {
 		return fmt.Errorf("create state reader: %v", err)
 	}
-	header, err := api._blockReader.Header(ctx, dbtx, hash, blockNumber)
+	header, err := api.headerByRPCNumber(ctx, rpc.BlockNumber(blockNumber), dbtx)
```

Comment:
FYI: in-theory `headerByHash` can use `it.HeaderNoCopy()` instead of `it.Header()`